addons/mrp/models/mrp_production.py

```diff
@@ -1231,7 +1231,9 @@ def _set_qty_producing(self, pick_manual_consumption_moves=True):
 
             new_qty = float_round((self.qty_producing - self.qty_produced) * move.unit_factor, precision_rounding=move.product_uom.rounding)
             move._set_quantity_done(new_qty)
-            if not move.manual_consumption or pick_manual_consumption_moves:
+            if self._context.get('mrp_display', False) and move.manual_consumption and pick_manual_consumption_moves:
```

Comment:
Why do you check on `pick_manual_consumption_moves` too?

Reply:
Hello @mohammadhossam,

We have observed that when producing MO from the backend, the pick_manual_consumption_moves value is set to false. However, when using the shop floor module, this value is set to true. Therefore, it is necessary to check.

Thank you.

Reply:
I think it's not set to true explicitly, but just the default value in the function definition is used. Also this opposes the expected meaning of the parameter. From its name, the expected behavior is to pick when it's true and vice versa, and this is what already happens in the next condition. But anyways, I'll leave this nit for the final reviewer to decide what is best.